docs/elasticsearch.rst

```diff
@@ -1,6 +1,56 @@
 ElasticSearch
 =============
 
+Playing Nice with ES: Simple things to know to get reasonable performance
```

Comment:
oh, good to know. I'm going to leave it as it is then. and just add to the bottom of `Querying ElasticSearch`.